app/src/main/java/com/github/se/cyrcle/ui/theme/molecules/ReportScreenMolecules.kt

```diff
@@ -0,0 +1,128 @@
+package com.github.se.cyrcle.ui.theme.molecules
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.github.se.cyrcle.R
+import com.github.se.cyrcle.model.parking.ImageReportReason
+import com.github.se.cyrcle.model.parking.ParkingReportReason
+import com.github.se.cyrcle.model.report.ReportedObjectType
+import com.github.se.cyrcle.model.review.ReviewReportReason
+import com.github.se.cyrcle.ui.navigation.NavigationActions
+import com.github.se.cyrcle.ui.report.ReportScreenAlertDialog
+import com.github.se.cyrcle.ui.theme.atoms.BulletPoint
+import com.github.se.cyrcle.ui.theme.atoms.ConditionCheckingInputText
+import com.github.se.cyrcle.ui.theme.atoms.Text
+import com.github.se.cyrcle.ui.theme.disabledColor
+
+@Composable
+fun ReportTopAppBar(navigationActions: NavigationActions, title: String) {
+  TopAppBar(navigationActions, title = title)
+}
+
+@Composable
+fun ReportTextBlock(title: String, bulletPoints: List<String>, modifier: Modifier = Modifier) {
+  Column(modifier = modifier.fillMaxWidth(), verticalArrangement = Arrangement.spacedBy(8.dp)) {
+    Text(
+        text = title,
+        style =
+            MaterialTheme.typography.titleMedium.copy(
+                fontWeight = FontWeight.Bold, textAlign = TextAlign.Start),
+        modifier = Modifier.fillMaxWidth().padding(vertical = 8.dp))
+    Column(
+        modifier = Modifier.padding(start = 16.dp, bottom = 16.dp),
+        verticalArrangement = Arrangement.spacedBy(8.dp)) {
+          bulletPoints.forEachIndexed { index, point ->
+            BulletPoint(point, testTag = "BulletPoint$index")
+          }
+        }
+  }
+}
+
+@Composable
+fun ReportInputs(
+    selectedReasonIfParking: MutableState<ParkingReportReason>? = null,
+    selectedReasonIfReview: MutableState<ReviewReportReason>? = null,
+    selectedReasonIfImage: MutableState<ImageReportReason>? = null,
```

Comment:
Arguments with default values should be placed last in the argument list

Reply:
didn't know that was a convention!